aspnetcore/fundamentals/servers.md

```diff
@@ -20,8 +20,8 @@ An ASP.NET Core app runs with an in-process HTTP server implementation. The serv
 ASP.NET Core ships with the following:
 
 * [Kestrel server](xref:fundamentals/servers/kestrel) is the default, cross-platform HTTP server.
-* IIS HTTP Server (`IISHttpServer`) is an [IIS in-process server](xref:fundamentals/servers/aspnet-core-module#in-process-hosting-model) implementation used with the [ASP.NET Core Module](xref:fundamentals/servers/aspnet-core-module).
-* [HTTP.sys server](xref:fundamentals/servers/httpsys) is a Windows-only HTTP server based on the [HTTP.sys kernel driver and HTTP Server API](https://msdn.microsoft.com/library/windows/desktop/aa364510.aspx). HTTP.sys is called [WebListener](xref:fundamentals/servers/weblistener) in ASP.NET Core 1.x.
+* IIS HTTP Server (`IISHttpServer`) is an [IIS in-process server](xref:host-and-deploy/aspnet-core-module#in-process-hosting-model) implementation used with the [ASP.NET Core Module](xref:host-and-deploy/aspnet-core-module).
```

Comment:
For consistency, can this start off with a link like the other two bullets?  Also it's not clear to me how putting IISHttpServer in code format/not a link is helpful here.

Also, shouldn't we have an h2 corresponding to this bullet, like we do for the first and third bullets?